src/locales/strings.js

```diff
@@ -0,0 +1,61 @@
+ // TODO(sjbarag): read from JS module generated from JSON files, probably.
+const translations = {
+  en: {
+    "hello": {
+      title: "hello, {name}",
+      note: "Says hi to a person.",
+    },
+    "usePersonalDuckAddr": {
+      title: "Use {email}",
+      note: "Shown when a user can choose their personal @duck.com address.",
+    },
+    "blockEmailTrackers": {
+      title: "Block email trackers",
+      note: "Shown when a user can choose their personal @duck.com address on native platforms.",
+    }
+  },
+  xa: {
+    "hello": {
+      title: "h33llllo, {name}",
+      note: "Says hi to a person.",
+    },
+    "usePersonalDuckAddr": {
+      title: "Ü55££ {email}",
+      note: "Shown when a user can choose their personal @duck.com address.",
+    },
+    "blockEmailTrackers": {
+      title: "Bl000ck €m@@@i1il1l träáåck33rr55",
+      note: "Shown when a user can choose their personal @duck.com address on native platforms.",
+    }
+  },
```

Comment:
Just a small pseudolocale for now. The structure of these objects (`title`+`note`) comes from content-scope-scripts' translation files.

Reply:
yep, looks good - we have the same in the privacy-dashboard too :)